library/Fission/CLI/IPFS/Connect.hs

```diff
@@ -28,25 +30,34 @@ swarmConnectWithRetry ::
   , MonadLocalIPFS m
   , MonadWebClient m
   )
-  => IPFS.Peer
+  => NonEmpty IPFS.Peer
   -> Int
   -> m (Either SomeException ())
-swarmConnectWithRetry _peer (-1) =
+swarmConnectWithRetry _peers (-1) =
   return . Left $ toException UnableToConnect
 
-swarmConnectWithRetry peer tries =
-  IPFS.Peer.connect peer >>= \case
-    Right _ ->
+swarmConnectWithRetry peers tries = do
+  attempts <- connectToPeers $ NonEmpty.toList peers
+  if any isRight attempts
```

Comment:
I almost want to know if any of them fail in an `infoDebug`. Thoughts?

Reply:
yeah that makes sense to me :+1: I'll just log the full connection results